acct-user/sndiod/sndiod-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="daemon user for sndio"
+ACCT_USER_ID="-1"
```

Comment:
Not allowed in ::gentoo. Please reserve a UID/GID in the [API](https://github.com/gentoo/api-gentoo-org) repo (pick a UID/GID unlikely to be taken for a bit and make a PR there, link it here).